neural_structured_learning/keras/adversarial_regularization.py

```diff
@@ -20,6 +20,8 @@
 import collections
 import functools
 import types
+from keras.utils import traceback_utils
+from keras.saving import save
```

Comment:
What is the earliest version of Keras providing these two modules (`save` and `traceback_utils`)? Please add a line to the [requirements file](https://github.com/tensorflow/neural-structured-learning/blob/master/neural_structured_learning/requirements.txt).